### ISSUE TITLE
Handling the Use of Two Resources Simultaneously in Azure OpenAI Dependency Injection

### ISSUE THREAD

**Issue by qq312888991** [betalgo/openai#267]
For example, I have deployed GPT3.5 and Dall-e in Azure OpenAI, with deployment names "gpt35turbo" and "delleV2" respectively. The dependency injection code in my Program.cs file is as follows, but it only allows me to use one of the deployed resources. How can I use the "delleV2" deployment instead?
```
services.AddOpenAIService(settings =>
  {
      settings.ApiKey = builder.Configuration["OpenAPI:ApiKey"];
      settings.ProviderType = ProviderType.Azure;
      settings.ResourceName = builder.Configuration["OpenAPI:ResourceName"];
      settings.DeploymentId = “gpt35turbo”//How to coexist gpt35turbo and delleV2？？
  });
```

**Comment by kayhantolga** [comment]
Yes @Frogley, I think multi-instance dependency injection will be ready this week.

**Comment by qq312888991** [comment]
> What you need is multi-instance dependency injection. I believe Tolga is working on this, you can check it [here](https://github.com/betalgo/openai/pull/258)

Okay, I understand. Thank you

**Comment by qq312888991** [comment]
> Yes @Frogley, I think multi-instance dependency injection will be ready this week.

Okay, I understand. Thank you

**Comment by kayhantolga** [comment]
fixed in v7.0.0 https://github.com/betalgo/openai/pull/271
